arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/ar/ArcGISArView.kt

```diff
@@ -856,6 +873,16 @@ class ArcGISArView : FrameLayout, DefaultLifecycleObserver, Scene.OnUpdateListen
         ) == PackageManager.PERMISSION_GRANTED
     }
 
+    /**
+     * Checks [requestedPermissions] property for permission to determine if it has already been
+     * requested this session. [permissionHasBeenPermanentlyDenied] returns false if we have
+     * never requested such a permission. So it's best to check both of these functions.
+     *
+     * @since 100.6.1
+     */
+    private fun permissionHasBeenRequestedPreviously(permission: String): Boolean =
+        requestedPermissions.contains(permission)
```

Comment:
Could we just move this function into `permissionHasBeenPermanentlyDenied`? It looks like both functions will always be called together anyway. I.e. just have one function that checks both...